settings_test.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import unicode_literals
+import os
+
+
+BASE_DIR = os.path.dirname(__file__)
+APPSERVER = os.uname()[1]
+
+SITE_ID = 1
+INTERNAL_IPS = ('127.0.0.1', )
+ALLOWED_HOSTS = ['localhost', '127.0.0.1', ]
+APPEND_SLASH = True
+TIME_ZONE = 'UTC'
+USE_TZ = True
+SECRET_KEY = 'abc'
+
+INSTALLED_APPS = (
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sites',
+    'django.contrib.admin',
+    'django.contrib.sessions',
+    'secure_redis',
+)
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'db.sqlite',
+    }
+}
+
+ROOT_URLCONF = 'urls'
+
+MIDDLEWARE_CLASSES = (
+    'django.middleware.security.SecurityMiddleware',
```

Comment:
@GhalebKhaled Most of those middlewares doesn't need to be here. I think only CommonMiddleware should stay here.